database/messages.js

```diff
@@ -1,69 +1,80 @@
 // connect to database
-require('./db');
+var db = require('./db');
 
 var mongoose = require('mongoose');
 var Promise = require('bluebird');
 mongoose.Promise = Promise;
 var messages = {};
 
 var MessageSchema = new mongoose.Schema({
+	id: Number,
```

Comment:
The reason why I didn't include this field is because mongodb already assigns an objectid, which you can query.

Furthermore, I thought if one message content is unacceptable, any other message including the exact same content should be deleted.